resources/tag.py

```diff
@@ -39,24 +40,29 @@ def get(self):
                     schema:
                       type: array
                       items:
-                        $ref: '#/components/schemas/Tag'
+                        $ref: '#/components/schemas/TagItemResponse'
         """
-        tag_prefix = request.args.get('query')
-
-        if not tag_prefix or len(tag_prefix) == 0:
-            tags = []
-        else:
-            tag_prefix = tag_prefix.lower()
```

Comment:
`.lower` is now handled in schema right? And stripping the tag doesn't result in any existing, undeletable tags?

Reply:
It is handled in schema: now API should be more consistent.